src/components/transaction/TransactionTable.js

```diff
@@ -55,7 +55,8 @@ const TransactionTable = () => {
       transactions
     );
     if (dateOfInvalidTransaction) {
-      showError(dateOfInvalidTransaction);
+      let errorMessage = `Your transaction dated ${dateOfInvalidTransaction} has invalid input.`;
+      props.setErrorMessage(errorMessage);
```

Comment:
i still think `showErrorMessage()` would be more descriptive in that the intent is to show it, not just to "set it"

Reply:
but i would be open either way if you want to leave it

Reply:
Let's leave for now. I can see justification for both ways.